components/intro/IntroMilestone.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        <p class="achievement">{{ $t(achievement) }}</p>
+        <p class="number">{{ $t(achievementNumber) }}</p>
+        <p class="achievement text-2xl mb-2">
+            {{ $t(achievementUnit) }}
```

Comment:
The content passed in here has already translated.
```suggestion
        <p class="achievement">{{ achievement }}</p>
        <p class="number">{{ achievementNumber }}</p>
        <p class="achievement text-2xl mb-2">
            {{ achievementUnit }}
```

Reply:
Okay, I see, so the contents will be translated in the `Intro.vue` component first, is that correct?

Reply:
yes, which is already done by you actually.